.github/workflows/publish.yml

```
@@ -59,3 +59,65 @@ jobs:
         with:
           name: docker-image
           path: fiftyone.tar.gz
+
+  publish-docker-images:
+    needs: [build, test]
+    runs-on: ubuntu-latest
+    matrix:
```

actionlint (1.7.4)

unexpected key "matrix" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)